examples/gnrc_minimal/Makefile

```diff
@@ -7,10 +7,10 @@ BOARD ?= native
 # This has to be the absolute path to the RIOT base directory:
 RIOTBASE ?= $(CURDIR)/../..
 
-# Comment this out to disable code in RIOT that does safety checking
+# Set this to 1 to enable code in RIOT that does safety checking
 # which is not needed in a production environment but helps in the
 # development process:
-DEVELHELP ?= 1
+DEVELHELP ?= 0
```

Comment:
Did you also check whether `NDEBUG` further reduces the firmware size?

Reply:
Shouldn't this be set, when setting `DEVELHELP` to 0?

Reply:
You mean to `1`? In any case, I don't seem to find anything that relates to your comment in the code base..

Reply:
Me neither. Up until now, I thought `DEVELHELP=0` deactivates `assert()`s oO

Reply:
It doesn't give any size advantages (I first thought it did, but I was on the wrong branch, so I just deactivated all the `asserts()` with `DEVELHELP` still activated... the force push removed the change that enabled `NDEBUG`)